### PR TITLE
Committing from command line.

### PR DESCRIPTION
**Pull request by Keeeeeeeks**
Hope this looks good; didn't name the package anything.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "something-awesome",
+  "version": "1.0.0",
+  "description": "test for submitting pull requests in command line",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```
Comment (sscovil) [inline package.json:7]: We will be using the `mocha` library for tests.

**Review by sscovil — APPROVED** [review]
Looks good! 👍

**Comment by elizabethdecoud13-sudo** [comment]
[](url)

File: package.json (hunk 1 above)
Comment (elizabethdecoud13-sudo) [reply package.json:7]: Si

**Comment by elizabethdecoud13-sudo** [comment]
Bueno

**Pull request merged**